packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -13,6 +13,13 @@ import type { IKeyboardEvent, IMouseEvent, IPointerEvent } from "../Events/devic
 import { DeviceSourceManager } from "../DeviceInput/InputDevices/deviceSourceManager";
 import { EngineStore } from "../Engines/engineStore";
 
+declare module "../scene" {
+    interface Scene {
+        /** @hidden */
+        _registeredActions: number;
+    }
+}

```

Comment:
I'm not sure this is right. `actionManager` depends on this property and currently does not have a dependency on `inputManager`. Should it just be an internal public property on scene directly?